frontend/src/components/signup/signup.js

```diff
@@ -64,7 +64,7 @@ const Signup = () => {
         e.preventDefault();
         try {
             await axios.post(REG_URL,
-                { name: user, password, password_confirmation: matchPassword, email },
+                { name: user, password, password_confirmation: matchPassword, email, isAdmin: false },
```

Comment:
I don't think we need to pass `isAdmin` for this one, because every new user is a student and the `isAdmin ` column should be `0` as default value